x-pack/plugin/src/test/resources/rest-api-spec/test/ml/update_model_snapshot.yml

```diff
@@ -1,207 +1,3 @@
-setup:
```

Comment:
These tests are now covered in `org.elasticsearch.xpack.ml.integration.JobModelSnapshotCRUDIT.java`